### ISSUE TITLE
can i put the TS server on different from gameserver ? 

### ISSUE THREAD

**Issue by goodguydul** [Itokoyamato/TokoVOIP_TS3#178]
can i put the TS server on different from gameserver ? 
because i use 3 different servers for game, database and Ts server.

**Comment by chrisrogersukgb** [comment]
Yes just the IP addresses as needed in the relevant configuration files.
